libraries/AP_NavEKF2/AP_NavEKF2_Outputs.cpp

```diff
@@ -534,7 +534,7 @@ void  NavEKF2_core::getFilterGpsStatus(nav_gps_status &faults) const
     faults.flags.bad_horiz_drift    = gpsCheckStatus.bad_horiz_drift; // GPS horizontal drift is too large to start using GPS (check assumes vehicle is static)
     faults.flags.bad_hdop           = gpsCheckStatus.bad_hdop; // reported HDoP is too large to start using GPS
     faults.flags.bad_vert_vel       = gpsCheckStatus.bad_vert_vel; // GPS vertical speed is too large to start using GPS (check assumes vehicle is static)
-    faults.flags.bad_fix            = gpsCheckStatus.bad_fix; // The GPS cannot provide the 3D fix required
+    faults.flags.bad_fix            = gpsCheckStatus.bad_fix; // The GPS cannot provide the minimum fix required
```

Comment:
Please revert this and AP_NavEKF3_Outputs.cpp

Reply:
Done